webapp/src/app.tsx

```diff
@@ -2018,6 +2020,9 @@ export class ProjectView
             }).done();
     }
 
+    identityLoginComplete() {
+    }
```

Comment:
TODO comment?

Reply:
This is unused. Will remove.

Reply:
I take that back. We will need it later. Will add a TODO comment.